lua/rustaceanvim/shell.lua

```diff
@@ -1,11 +1,16 @@
 local M = {}
 
 ---@return boolean
-local function is_windows()
+function M.is_windows()
   local sysname = vim.loop.os_uname().sysname
   return sysname == 'Windows' or sysname == 'Windows_NT'
 end
 
+---@return boolean
+function M.is_macos()
+  return vim.loop.os_uname().sysname == 'Darwin'
```

Comment:
note to self (for another issue): `vim.loop` will be deprecated in Neovim 0.10 (replaced with `vim.uv`). Use `require('rustaceanvim.compat').uv` instead (the existing `vim.loop` calls are ones I forgot to fix when I forked the project).